examples/text_summarization/bertsum_cnndm.ipynb

```diff
@@ -0,0 +1,3921 @@
+{
```

Comment:
<p>What if I want to use multiple GPUs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>